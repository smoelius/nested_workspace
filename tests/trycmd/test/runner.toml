args = ["test", "--workspace"]

[bin]
name = "cargo"

[fs]
cwd = "../../../fixtures/runner"
