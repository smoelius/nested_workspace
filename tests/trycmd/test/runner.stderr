...
   Compiling runner v0.1.0 ([CWD])
    Finished `test` profile [unoptimized + debuginfo] target(s) in [..]s
     Running unittests src/lib.rs (target/debug/deps/runner-[..])
running: ["[..]/runner-[..]"]
<<< [CWD]/package
   Compiling package v0.1.0 ([CWD]/package)
    Finished `test` profile [unoptimized + debuginfo] target(s) in [..]s
     Running unittests src/lib.rs (target/debug/deps/package-[..])
running: ["[..]/package-[..]"]
>>> [CWD]/package
